recipes/biomechanical-analysis-framework/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "biomechanical-analysis-framework" %}
+{% set cxx_name = "libbiomechanical-analysis-framework" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
```

```suggestion
  # {{ name }}
  name: biomechanical-analysis-framework
```

```suggestion
  # {{ cxx_name }}
  - name: libbiomechanical-analysis-framework
```

See https://github.com/conda-forge/gz-sim-yarp-plugins-feedstock/pull/3 and the links there, we can't use the jinja macro in the name of the packages.